yt_dlp/extractor/fujitv.py

```diff
@@ -5,19 +5,19 @@
 
 
 class FujiTVFODPlus7IE(InfoExtractor):
-    _VALID_URL = r'https?://i\.fod\.fujitv\.co\.jp/plus7/web/[0-9a-z]{4}/(?P<id>[0-9a-z]+)'
+    _VALID_URL = r'https?://fod\.fujitv\.co\.jp/title/[0-9a-z]{4}/(?P<id>[0-9a-z]+)'
```

Comment:
why does this doesn't have any tests?

Reply:
don't break the existing valid_url. And would be nice if u can add some tests @YuenSzeHong